sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/xml/TestSuite.java

```diff
@@ -100,8 +106,8 @@ public TestSuite(final String name, final String file, final String tests, final
     this.tests = tests;
     this.assertions = assertions;
     this.time = time;
-    this.testSuites = testSuites;
-    this.testCases = testCases;
+    this.testSuites = CollectionUtils.isEmpty(testSuites) ? Collections.<TestSuite>emptyList() : testSuites;
+    this.testCases = CollectionUtils.isEmpty(testCases) ? Collections.<TestCase>emptyList() : testCases;
```

Comment:
Hi @vilchik-elena that was to prevent null testSuites or testCases. In case we are preventing these two objects of being null elsewhere, we can remove this change.